### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by Kutrim** [MichelNivard/gptstudio#120] [General]
Can someone help-me?
A cant to use the chatgpt. By excepting the command, it gives me no way out
![image](https://github.com/MichelNivard/gptstudio/assets/134450837/a94e40c4-7e6c-4c52-88f9-c3e352ca11a2)

![image](https://github.com/MichelNivard/gptstudio/assets/134450837/aa447b83-00b5-47e3-b99b-fdd448b5187f)

**Comment by calderonsamuel** [comment]
I think this was addressed in #100 . Please refer to that issue and, if the problem is different, open a new one with as much detail as possible.
